runtime-manager/modules/ROOT/pages/rtm-traffic-inspection.adoc

```diff
@@ -0,0 +1,111 @@
+* The Inspection Proxy server does not require client authentication from the agents. The proxy does not require the Mule Agent to present a client certificate. Communication between the Mule Agent and the inspection proxy is TLS, not mTLS.
+* The inspection proxy must be provisioned to send a customer-private certificate to the Mule Agent. The Mule Agent uses a Certificate Authority from the JVM's keystore to validate the public certificate presented by the inspection proxy.
+* The inspection proxy and the MuleSoft control plane communicate via mTLS. There are two certificates involved:
+** The control plane presents a MuleSoft public server certificate to the inspection proxy. The proxy must be provisioned with the correct Certificate Authority to validate the server certificate presented by the MuleSoft control plane
+** The control plane requires a client certificate from the inspection proxy. This certificate must be provided by the customer to MuleSoft. 
++
+[NOTE]
+Communication with the control plane fails if the certificate does not match the specified serial number and common name.
+
+== Mule Installation and Registration Steps
+
+=== Provision the Truststore of the JVM with the Proxy Root CA
+
+. Identify the folder location of the Java Virtual Machine.
+. Insert the Root CA of the proxy in the truststore of the mule's JVM: in your terminal window, run the following command, replacing `$JAVA_HOME` with the actual path:
++
+[source,console,linenums]
+----
+sudo keytool -import -alias testCert -keystore $JAVA_HOME/jre/lib/security/cacerts -file proxy_cacert.pem
+----
++
+. Enter the provided password.
+. If you have multiple versions of Java, insert the certificate in the version that is being used by the Mule runtime.
+
+[[install-mule-runtime]]
+=== Install the Mule Runtime
+
+. Install the latest available Mule runtime version. You can skip this step if already installed.
++
+To check the latest Mule Runtime version, see xref:release-notes::mule-runtime/mule-esb.adoc[].
++
+[NOTE] 
+The Mule runtime installation bundle includes both the Mule Runtime Engine and the Mule Runtime Agent. 
+
+=== Upgrade the Mule Runtime Agent
+
+Make sure that the version of the Mule Runtime agent is `2.5.6` or later. If you have an earlier version, update the runtime agent before registering.
+
+To update the Runtime agent:
+
+. Download the agent's zip file
+. Extract the downloaded `agent-setup-2.5.6.zip` file to `$MULE_HOME/bin`.
++
+If prompted, overwrite any conflicting files.
+. Do NOT run `amc_setup -U`.
+
+=== Register the Mule Runtime
+
+. Update the file `wrapper.conf` file with the IP and port of the traffic inspection proxy by following instructions in xref:rtm-agent-proxy-config.adoc#set-up-proxy-server-configuration-in-the-wrapper-conf-file[Set Up Proxy Server Configuration in the wrapper.conf File].
+. Login in to the Anypoint console.
+. From the Anypoint Platform, select *Runtime Manager*.
+. Click *Servers* in the navigation menu.
```

Comment:
We use "select" for menu choices (and "click" for buttons).
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22select,%22%22choose%22